packages/venia-ui/lib/components/ProductSort/sortItem.js

```diff
@@ -11,24 +11,32 @@ const SortItem = props => {
     const { active, onClick, sortItem } = props;
     const classes = useStyle(defaultClasses, props.classes);
 
-    const handleClick = useCallback(() => {
-        onClick(sortItem);
-    }, [sortItem, onClick]);
+    const handleClick = useCallback(
+        e => {
+            if (e.button === 0) {
+                onClick(sortItem);
+            }
+        },
+        [sortItem, onClick]
+    );
 
-    const handleKeyDown = e => {
-        if (e.key === 'Enter' || e.key === ' ') {
-            e.preventDefault();
-            handleClick();
-        }
-    };
+    const handleKeyDown = useCallback(
+        e => {
+            if (e.key === 'Enter' || e.key === ' ') {
+                e.preventDefault();
+                onClick(sortItem);
+            }
+        },
+        [onClick, sortItem]
+    );
 
     const activeIcon = active ? <Icon size={20} src={Check} /> : null;
 
     return (
         <button
             className={classes.root}
             data-cy={active ? 'SortItem-activeButton' : 'SortItem-button'}
-            onMouseDown={handleClick}
+            onMouseDown={e => handleClick(e)}
```

Comment:
Doesn't `onMouseDown={handleClick}` already send the event?

Reply:
You're right, I'll fix it.